utils/experiment-generator.py

```diff
@@ -0,0 +1,196 @@
+# To use this code, make sure you
```

Comment:
Let's move this inside `utils/experiment-examples/` so it remains beside the file it generates.